app/static/src/app/components/fit/FitTab.vue

```diff
@@ -1,14 +1,20 @@
 <template>
   <div class="fit-tab">
     <div>
-      <button class="btn btn-primary" id="fit-btn" :disabled="!canFitModel" @click="fitModel">Fit model</button>
+      <button class="btn btn-primary me-2" id="fit-btn" :disabled="!canFitModel" @click="fitModel">Fit model</button>
+      <button class="btn btn-primary" id="cancel-fit-btn" :disabled="!fitting" @click="cancelFit">Cancel fit</button>
```

Comment:
Could make the cancel button secondary, outline, or danger, to differentiate it from the fit/action button